tools/ci_build/github/azure-pipelines/templates/jobs/win-ci-vs-2022-job.yml

```diff
@@ -96,6 +96,18 @@ jobs:
       WithCache: ${{parameters.WITH_CACHE}}
       Today: $(Today)
 
+  - ${{ if eq(parameters.isX86, true) }}:
+    - task: UseDotNet@2
+      inputs:
+        version: '7.x'
+      env:
+        PROCESSOR_ARCHITECTURE: 'x86'
+
+  - ${{ if contains(parameters.additionalBuildFlags, 'use_cuda') }}:
+    - task: UseDotNet@2
+      inputs:
+        version: '7.x'
```

Comment:
Would it be simpler to instead always have a UseDotNet@2 task, with an 'if' to set processor architecture, and an 'else' to not?